### PR TITLE
Updates from building larsoft 09-93-00.

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Review by knoepfel — COMMENTED** [review]
@gartung, why do these packages need python dependencies?

**Comment by gartung** [comment]
When I was using freeze these packages lost their indirect  dependence on python and so had not PYTHON_PATH set for ROOT(?) and cppyy was not found.

But I should reverse these changes since we are not using ROOT as an external.

File: packages/genie/package.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def url_for_version(self, version):
     patch("patch/sles-cnl.patch", when="platform=cray")
     patch("patch/root_subdir.patch")
 
-    patch("patch/GENIE-Generator.patch", when="@3.04.02")
+#    patch("patch/GENIE-Generator.patch", when="@3.04.02")
```
Comment (knoepfel) [inline packages/genie/package.py]: Suggest removing the line entirely.

Comment (knoepfel) [reply packages/genie/package.py]: Based on SciSoft discussion this line should be kept, but with `when="@3.04.00"`.

**Review by marcmengel — COMMENTED** [review]
Looks good.

**Pull request merged**